src/ensembl/src/content/app/browser/browser-location-indicator/BrowserLocationIndicator.scss

```diff
@@ -0,0 +1,53 @@
+/* stylelint-disable no-descending-specificity */
```

Comment:
Stylelint was complaining about this file because of the order in which the classes are defined:

![image](https://user-images.githubusercontent.com/6834224/71809514-255ff880-3068-11ea-9ffe-801ce0940c08.png)

There are several possible solutions:
1) define `chrCode` and `chrRegion` classes above the `browserLocationIndicator` class
2) nest definitions of `chrCode` and `chrRegion` classes inside the `browserLocationIndicator` class
3) disable the rule just for this file
4) disable the rule globally, by adding a stylelint config file

I don't really like solution 1, because `browserLocationIndicator` class is the class for the whole component and, intuitively, should be the first class that developer reads about.
Solution 2 is a bit better, but still annoying, because, technically, there is absolutely no need to nest classes of child elements within the parent class (and get css rules that say `.browserLocationIndicator .chrCode` rather than just `.chrCode`. We only need to override css rules for elements inside of a disabled component.
Not sure about solutions 3 or 4. Solution 3 is the simplest, but does not prevent the linter from similar complaints later on; solution 4 will stop this rule from annoying us entirely.